active/0001-automated-releases.md

```diff
@@ -0,0 +1,97 @@
+- Automated Releases
+- @kleimkuhler, @alpeb
+- 2020-02-17
+- 2020-03-03
```

Comment:
Can you add some titles/labeling to these lines?

Reply:
@alpeb Would "beginning of sprint" and "end of sprint" be helpful? I agree the dates by themselves aren't too helpful.